test/TestUtilities/Test.Utility/TestFallbackCertificateBundleX509ChainFactory.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using NuGet.Common;
+using NuGet.Packaging.Signing;
+
+namespace Test.Utility.Signing
+{
+    internal sealed class TestFallbackCertificateBundleX509ChainFactory
+#if NET5_0_OR_GREATER
+        : CertificateBundleX509ChainFactory
+#endif
+    {
+        internal static TestFallbackCertificateBundleX509ChainFactory Instance { get; } = new();
+
+        private TestFallbackCertificateBundleX509ChainFactory()
+#if NET5_0_OR_GREATER
+            : base(LoadCertificates())
+#endif
+        {
+        }
+
+#if NET5_0_OR_GREATER
+        private static X509Certificate2Collection LoadCertificates()
+        {
+            byte[] bytes = SigningTestUtility.GetResourceBytes("codesignctl.pem");
```

Comment:
What's the expiration date of this cert?  Will tests start failing once this cert expires?

Reply:
There are 258 certificates in the file.  Currently, 219 are valid, and 39 are expired.  Tests will not start failing when a certificate expires any more than they would if a certificate in Windows' trusted root authority certificates store expired.